actionlist/coverage/clover.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@burnumd should the coverage files be pushed into the repo? Can we just do this locally and ignore it?